### ISSUE TITLE
Failure when starting new pod after upgrade 

### ISSUE THREAD

**Issue by scottf51** [LoopKit/Loop#1179]
After updating to the latest dev (as of last night ) , I got errors when I tried to replace pod 

sorry if I miss some detail but I can't find the guide that tells me everything I should include for some reason and it's not showing the skeleton outline on here 

pump: Omnipod eros
iOS : 13.2.2
loop branch - dev
version : listed as 1.10.1 - but that doesn't get bumped , I built it from the source approx 11 hours ago 

steps to reproduce : 

1. upgrade loop with pod already paired 

2. after x hours replace pod

3. disable pod , looks ok

4. pair new pod and get "pod fault: fault event code 0x00: no fault "

5. try again and got a response of "." 

6. Finally let it "deactivate| the pod , didn't seem to do anything , but then I was able to add the same pod again as new.

I did have a few issues with "cannot communicate with pod" recently before the upgrade, but assumed that was just a bad pod and never got an actual fault.

I will attach a screenshot and a log capture , log capture was from after I finally got it paired.

**Comment by scottf51** [comment]
[Loop Report 2019-11-16 11:21:07Z.zip](https://github.com/LoopKit/Loop/files/3854126/Loop.Report.2019-11-16.11.21.07Z.zip)
![IMG_0576](https://user-images.githubusercontent.com/45606048/68992546-bda9e000-0864-11ea-9cc3-bf3dd0d34794.png)

**Comment by rhinographix** [comment]
I am also having this issue.

pump: Omnipod Eros
iPhone 8
iOs: 13.2.3
Loop Branch: dev v1.10.1
buildDateString: Wed Nov 20 22:14:07 CST 2019
xcodeVersion: 11A420a

I switched to Dev branch (from Omnipod-testing) last night, and tonight is the first time I've tried pairing from this branch.

At first, I got the error that no pod was in range. Stuck pod in microwave walked away. Canceled pod. Walked back and opened microwave. Tried to pair again and got the single “.” 

I’ve tried the pod in microwave several times while resetting on phone. Resetting RileyLink. Closing both Loop and Omnipod G6 apps. Restarting phone. Disconnecting and reconnecting RL from Loop, green light on RileyLink turns on. 

Still keep getting this error. 

This seems to happen when I’m trying to switch pods before I go to bed... not related... just tired.

![IMG_8403](https://user-images.githubusercontent.com/53888192/69406309-6a51ea80-0cc7-11ea-91fe-dca202ad4ee8.PNG)
[Loop Report 2019-11-22 01:12:04-06:00.md.zip](https://github.com/LoopKit/Loop/files/3878385/Loop.Report.2019-11-22.01.12.04-06.00.md.zip)
